src/frontend/src/pages/Calendar/CalendarComponents/MonthSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, MenuItem, TextField } from '@mui/material';
+import { Dispatch, SetStateAction } from 'react';
+import { EnumToArray, MONTH_NAMES } from '../../../utils/design-review.utils';
+
+interface MonthSelectorProps {
+  displayMonth: Date;
+  setDisplayMonth: Dispatch<SetStateAction<Date>>;
+}
+
+const MonthSelector: React.FC<MonthSelectorProps> = ({ displayMonth, setDisplayMonth }) => {
```

Comment:
Can you put a little space between the two selects (maybe also see how changing their background color looks)
<img width="183" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/de411c9f-6edc-4e3a-af4a-ba5440bbafa2">